core/zms/src/main/rdl/Domain.rdli

```diff
@@ -282,3 +282,18 @@ resource DomainTemplate DELETE "/domain/{name}/template/{template}" {
         ResourceError TOO_MANY_REQUESTS;
     }
 }
+
+//List of Business Services
+type BusinessServiceList Struct {
```

Comment:
I don't see why we need to define a specific object type for Business Services. This is not an Athenz object - just list of strings so we should keep at as is - Array<String>, more details on the api call itself

Reply:
I’m fixing it so the endpoint will be used to fetch valid values for attribute but I still won’t be able to return a list of strings as you showed in the example:

`resource Array<String> GET "/domain/metastore?attribute=<attributeName>&user=<userName>
`
RDL generator doesn’t accept it. The resource must be a declared type even if it’s just a String array (the name of the function generated in ZMSResources is determined by this type + action) So I’ve created the following type:
```
//List of valid domain meta attribute values
type DomainMetaStoreValidValuesList Struct {
    Array<String> validValues; //list of valid values for attribute
}
```